ocs_ci/helpers/helpers.py

```diff
@@ -4480,3 +4488,111 @@ def verify_log_exist_in_pod_logs(
     )
     logger.info(f"logs osd:{pod_logs}")
     return expected_log in pod_logs
+
+
+def retrieve_cli_binary(cli_type="mcg"):
+    """
+    Download the MCG-CLI/ODF-CLI binary and store it locally.
+
+    Args:
+        cli_type (str): choose which bin file you want to download ["odf" -> odf-cli , "mcg" -> mcg-cli]
+
+    Raises:
+        AssertionError: In the case the CLI binary is not executable.
+
+    """
+    semantic_version = version.get_semantic_ocs_version_from_config()
+    if cli_type == "odf" and semantic_version < version.VERSION_4_15:
+        raise NotSupportedException(
+            f"odf cli tool not supported on ODF {semantic_version}"
+        )
+
+    remote_path = get_architecture_path(cli_type)
+    remote_cli_basename = os.path.basename(remote_path)
+    if cli_type == "mcg":
+        local_cli_path = constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH
+    elif cli_type == "odf":
+        local_cli_path = constants.CLI_TOOL_LOCAL_PATH
+    local_cli_dir = os.path.dirname(local_cli_path)
+
+    if (
+        config.DEPLOYMENT["live_deployment"]
+        and semantic_version >= version.VERSION_4_15
+    ):
+        image = f"{constants.ODF_CLI_OFFICIAL_IMAGE}:v{semantic_version}"
+    elif (
+        config.DEPLOYMENT["live_deployment"]
+        and semantic_version >= version.VERSION_4_13
+    ):
+        image = f"{constants.MCG_CLI_IMAGE}:v{semantic_version}"
+
+    elif semantic_version >= version.VERSION_4_15:
+        image = f"{constants.ODF_CLI_DEV_IMAGE}:v{semantic_version}"
+    else:
+        image = f"{constants.MCG_CLI_IMAGE_PRE_4_13}:{get_ocs_build_number()}"
```

Comment:
```suggestion
    live_deployment = config.DEPLOYMENT["live_deployment"]
    if and semantic_version >= version.VERSION_4_15:
        if live_deployment: 
            image = f"{constants.ODF_CLI_OFFICIAL_IMAGE}:v{semantic_version}"
        else:
            image = f"{constants.ODF_CLI_DEV_IMAGE}:v{semantic_version}"
    elif semantic_version >= version.VERSION_4_13:
        if live_deployment:
            image = f"{constants.MCG_CLI_IMAGE}:v{semantic_version}"
        else:
            image = f"{constants.MCG_CLI_IMAGE_PRE_4_13}:{get_ocs_build_number()}"
    else:
        image = f"{constants.MCG_CLI_IMAGE_PRE_4_13}:{get_ocs_build_number()}"
```

I meant something like this

Reply:
From some reason the MCG_CLI image is called: MCG_CLI_IMAGE_PRE_4_13 but it's still being built also for 4.15. So maybe rename it as DEV image like you did for ODF_CLI constant but we need to check if we don't refer it anywhere.

Reply:
done